test/appium/tests/atomic/chats/test_activity_center.py

```diff
@@ -0,0 +1,251 @@
+import pytest
+
+from tests import marks
+from tests.base_test_case import MultipleSharedDeviceTestCase, create_shared_drivers
+from views.sign_in_view import SignInView
+
+@pytest.mark.xdist_group(name="activity_center_medium_2")
+@marks.medium
+class TestActivityCenterMultipleDeviceMedium(MultipleSharedDeviceTestCase):
+
+    @classmethod
+    def setup_class(cls):
+        cls.drivers, cls.loop = create_shared_drivers(2)
+        cls.device_1, cls.device_2 = SignInView(cls.drivers[0]), SignInView(cls.drivers[1])
+        cls.home_1, cls.home_2 = cls.device_1.create_user(), cls.device_2.create_user()
+        cls.public_key_user_1, cls.username_1 = cls.home_1.get_public_key_and_username(return_username=True)
+        cls.public_key_user_2, cls.username_2 = cls.home_2.get_public_key_and_username(return_username=True)
+        [cls.group_chat_name_1, cls.group_chat_name_2, cls.group_chat_name_3, cls.group_chat_name_4, \
+         cls.group_chat_name_5] = "GroupChat1", "GroupChat2", "GroupChat3", "GroupChat4", "GroupChat5"
+        cls.message_from_sender = "Message sender"
+        cls.home_2.home_button.double_click()
+        cls.device_2_one_to_one_chat = cls.home_2.add_contact(cls.public_key_user_1)
+
+    @marks.testrail_id(702183)
+    def test_activity_center_reject_chats(self):
+
+        self.device_2.just_fyi('Device2 sends a message in 1-1 chat to Device1')
+        self.device_2_one_to_one_chat.send_message(self.message_from_sender)
+
+        [home.home_button.double_click() for home in [self.home_1, self.home_2]]
+
+        self.device_2.just_fyi('Device2 creates group chat with Device1')
+        self.home_2.create_group_chat([self.username_1], group_chat_name=self.group_chat_name_1)
+        self.home_2.home_button.double_click()
+        self.home_1.home_button.double_click()
+
+        self.device_1.just_fyi('Device1 rejects both chats and verifies they disappeared and not in Chats too')
+        self.home_1.notifications_button.click()
```

Comment:
Better to replace here self.home_1.notifications_unread_badge.wait_and_click(wait_time) as it may be that chats won't appear quickly and appium may not find them

Reply:
done